cli/connector/opts.go

```diff
@@ -66,7 +68,13 @@ func MakeConnectOpts(connString, username, password string,
 
 	addrLen := len(address)
 	switch {
-	case addrLen > 0 && (address[0] == '.' || address[0] == '/'):
+	case addrLen > 0 && (address[0] == '.' || address[0] == '/' || address[0] == '~'):
```

Comment:
Extend unit tests for MakeConnectOpts to check these changes.

Reply:
Unit tests extended.